.circleci/config.yml

```diff
@@ -170,7 +215,7 @@ jobs:
 workflows:
   hourly:
     jobs:
-      - golang-test:
+      - golang-validate-all:
```

Comment:
I think we'll probably want an additional hourly job, instead of replacing the existing one:
1. We can leave the existing one that runs `golang-test` and sends failures to `notify-ci-failures` (which is how we get notifications if something is wrong with our code)
2. Create the new `golang-validate-all` job, as you have done here, which sends failures to `notify-chain-validate-failures`, indicating there is an issue with an external chain

Reply:
Sounds good, i've now got both `golang-test` and `golang-validate-all` to run hourly, with notifications going to separate slack channels.